internal/provider/clickhouse_cluster_resource_test.go

```diff
@@ -24,41 +26,77 @@ func TestAccClickhouseClusterResource(t *testing.T) {
 		RegionId:  types.StringValue("eu-central-1"),
 		CloudType: types.StringValue("aws"),
 		NetworkId: types.StringValue(testNetworkId),
-
 		Resources: &clickhouseClusterResources{
 			Clickhouse: &clickhouseClusterResourcesClickhouse{
-				ResourcePresetId: types.StringValue("s1-c2-m4"),
+				ResourcePresetId: types.StringValue("s2-c2-m8"),
```

Comment:
Oh, glad you want to change it.
Could you change it to `g2-c2-m8` for tests? Just cheaper and arm64 ftw.

Reply:
Ok